opensrp-reveal/src/test/java/org/smartregister/reveal/util/TestingUtils.java

```diff
@@ -85,4 +93,19 @@ public static Task getTask(String entityId) {
         return task;
     }
 
+
+    public static com.mapbox.geojson.Feature getStucture() {
+        return com.mapbox.geojson.Feature.fromJson(structureJSON);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.mapbox.geojson.Feature.fromJson' due to existing import 'com.mapbox.geojson.Feature'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)